Move contributing guide details to website and rename two sections (#1335)
GenericMappingTools/pygmt: doc/contributing.md

# Contributors Guide

This is a community driven project and everyone is welcome to contribute.

The project is hosted at the
[PyGMT GitHub repository](https://github.com/GenericMappingTools/pygmt).

The goal is to maintain a diverse community that's pleasant for everyone.
**Please be considerate and respectful of others**. Everyone must abide by our
[Code of Conduct](https://github.com/GenericMappingTools/pygmt/blob/master/CODE_OF_CONDUCT.md)
and we encourage all to read it carefully.

## Ways to Contribute

* Tackle any issue that you wish! Some issues are labeled as **"good first issues"** to
  indicate that they are beginner friendly, meaning that they don't require extensive
  knowledge of the project.
* Make a tutorial or gallery example of how to do something.
* Provide feedback about how we can improve the project or about your particular use
  case.
* Contribute code you already have. It doesn't need to be perfect! We will help you
  clean things up, test it, etc.

## Getting Help

> Discussion often happens in the issues and pull requests.

Reviewer: ```suggestion
Discussion often happens on GitHub issues and pull requests.
```
